README.md

```diff
@@ -4,6 +4,12 @@
 
 Quire is a modern, multi-format publishing framework designed to create books as authoritative and enduring as print, and as vibrant and feature-rich as the web, from a single set of plain text files. All without ever signing-up for an account, paying a fee, or setting up and maintaining a complicated server.
 
+Quire is a tool for publishing professionals and private individuals with an interest in digital publishing and everyone in between. Quire’s versatility lends itself to a variety of projects and users of all technology levels.
```

Comment:
Re-reading this now, I think "users of all technology levels" isn't quite accurate. There are more technical hurdles to working with Quire (at least in it's current state) than the average user might be comfortable. Can you reword?